hugegraph-api/src/main/java/com/baidu/hugegraph/auth/StandardAuthenticator.java

```diff
@@ -65,17 +65,25 @@ private void initAdminUser() throws Exception {
     }
 
     private String inputPassword() {
+        String password = "";
         String prompt = "Please input the admin password:";
+        String alertMsg = "password must be not empty";
```

Comment:
String prompt => inputPrompt
String alertMsg => String notEmptyPrompt
improve message "The admin password can't be empty"